ambry-api/src/test/java/com.github.ambry/rest/RestUtilsTest.java

```diff
@@ -556,23 +557,30 @@ public void buildGetBlobOptionsTest()
     doBuildGetBlobOptionsTest("bytes=-8", ByteRange.fromLastNBytes(8), true, false);
     doBuildGetBlobOptionsTest("bytes=-123456789", ByteRange.fromLastNBytes(123456789), true, false);
     // bad ranges
-    String[] badRanges = {"bytes=0-abcd", "bytes=0as23-44444444",
-        "bytes=22-7777777777777777777777777777777777777777777", "bytes=22--53", "bytes=223-34", "bytes=-34ab",
-        "bytes=--12", "bytes=-12-", "bytes=12ab-", "bytes=---", "btes=3-5", "bytes=345", "bytes=3.14-22",
-        "bytes=3-6.2", "bytes=", "bytes=-", "bytes= -"};
+    String[] badRanges =
+        {"bytes=0-abcd", "bytes=0as23-44444444", "bytes=22-7777777777777777777777777777777777777777777", "bytes=22--53", "bytes=223-34", "bytes=-34ab", "bytes=--12", "bytes=-12-", "bytes=12ab-", "bytes=---", "btes=3-5", "bytes=345", "bytes=3.14-22", "bytes=3-6.2", "bytes=", "bytes=-", "bytes= -"};
```

Comment:
the formatter seems to want to it this way, but I'll break it into multiple lines.